src/Index.php

```diff
@@ -272,10 +275,7 @@ public function getDocument($id, array $options = []): Document
             $data = [];
         }
 
-        $document = new Document($id, $data, $this->getName());
-        $document->setVersion($result['_version']);
```

Comment:
Not sure this should be removed finally. As it's still documented as returned.

Reply:
I'll put back setVersion, just remove it for a document update